cmd/ocm-support/get/clusters/cmd.go

```diff
@@ -0,0 +1,75 @@
+package clusters
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/cluster"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	first bool
+}
+
+// Define the command structure
+var CmdGetClusters = &cobra.Command{
+	Use:     "clusters [org_id]",
+	Aliases: utils.Aliases["clusters"],
```

Comment:
In addition to the org_id, I think we should support the same parameters as subscriptions:

 [id|external_id|organization_id] [optional_search_string]

Reply:
Updated it to use:
`clusters [id|external_id|organization_id|subscription_id] [optional_search_string]`

- id = cluster_id
- external_id = cluster_external_id